src/operator/nn/activation-inl.h

```diff
@@ -168,6 +173,10 @@ void ActivationGradComputeImpl(const ActivationParam &param, const OpContext &ct
         ActivationBackward<xpu, mshadow_op::softrelu, mshadow_op::softrelu_grad, DType>(
             ctx, out_grad, out_data, req, output);
         break;
+      case activation::kSoftSign:
```

Comment:
Are you sure this is correct? 

`ActivationGradComputeImpl` takes `out_grad` and `out_data` to calculate `output`. In another word, 
for `y = activation(x)`, it calculates `dx = _backward_activation(dy, y)`, not `dx = _backward_activation(dy, x)`.